concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1679,15 +1679,15 @@ Hale-1973-1798-1677	Attributes-152	rough	adj	12c.29	1923	ROUGH
 Hale-1973-1798-1678	Attributes-153	solid	adj	12c.30	3003	SOLID
 Hale-1973-1798-1679	Attributes-154	hollow	adj	12c.31	3701	HOLLOW
 Hale-1973-1798-1680	Attributes-155	empty	adj	12c.32	1624	EMPTY
-Hale-1973-1798-1681	Attributes-156	open	adj	12c.33		
-Hale-1973-1798-1682	Attributes-157	shut	adj	12c.34		
+Hale-1973-1798-1681	Attributes-156	open	adj	12c.33	3235	OPEN (STATE)
+Hale-1973-1798-1682	Attributes-157	shut	adj	12c.34	2063	CLOSE (EYES)
```

Comment:
I'd rather map this to `589 CLOSED` https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L590